katportalclient/client.py

```diff
@@ -71,13 +71,13 @@ def create_jwt_login_token(email, password):
     return jwt_auth_token
 
 
-class SensorSample(namedtuple('SensorSample', 'timestamp, value, status')):
+class SensorSample(namedtuple('SensorSample', 'sample_time, value, status')):
     """Class to represent all sensor samples.
 
     Fields:
-        - timestamp:  float
-            The timestamp (UNIX epoch) the sample was received by CAM.
-            timestamp value is reported with millisecond precision.
+        - sample_time:  float
+            The sample_time (UNIX epoch) the sample was received by CAM.
```

Comment:
Suggest "The time ...", rather than "The sample_time ...".
Same on lines 101, 104 and other places where we are referring to time in general, not the field name.